docs/UserGuide.md

```diff
@@ -118,47 +118,95 @@ Format: `project -d PROJECT_ID`
 Examples:
 * `project -d 1` Deletes project with `PROJECT_ID` 1.
 
-### Tagging a client : `client -t`
+### Finding a project : `project -f`
 
-Tags a client to a project.
+Finds and lists all the projects matching the search criteria.
 
-Format: `client -t PROJECT_ID n/CLIENT_NAME [p/CLIENT_CONTACT_NUMBER] [e/CLIENT_EMAIL]…​`
+Format: `project -f [n/PROJECT_NAME] [r/REPOSITORY]…​`
 
-* Adds the client to the project with the specified `PROJECT_ID`.
+* Finds all the projects with the specified `PROJECT_NAME` and `REPOSITORY`.
+* Finds all the projects such that the fields under the project contain at least one word from the keywords provided
+  after each search criteria.
+* The keywords provided must be valid arguments for their respective search criteria.
 
 Examples:
-*  `client -t 1 n/Amy` Tags the project with `PROJECT_ID` 1 with a client with `CLIENT_NAME` Amy.
-*  `client -t 2 n/Bob p/91234567` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` Bob and `CLIENT_CONTACT_NUMBER` 91234567.
-*  `client -t 3 n/Charlie e/charlie@gmail.com` Tags the project with `PROJECT_ID` 3 with a client with `CLIENT_NAME` Charlie and `CLIENT_EMAIL` charlie@gmail.com.
-*  `client -t 2 n/Dave p/91111111 e/dave@gmail.com` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` Dave, `CLIENT_CONTACT_NUMBER` 91111111 and `CLIENT_EMAIL` dave@gmail.com
+* `project -f n/DevEnable` Finds and lists all the projects whose `PROJECT_NAME` contains the word DevEnable.
+* `project -f n/DevEnable AB3` Finds and lists all the projects whose `PROJECT_NAME` contains the word DevEnable or AB3.
+* `project -f r/tp/F13` Finds and lists all the projects with `REPOSITORY` tp/F13.
+* `project -f n/AB4 AB3 r/tp/F13` Finds and lists all the projects whose `PROJECT_NAME` contains the word AB4 or AB3 
+  and with `REPOSITORY` tp/F13.
+
+### Adding a client : `client -a`
+
+Adds a client to the list of clients and to the specified project.
+
+Format: `client -a n/CLIENT_NAME pid/PROJECT_ID [p/CLIENT_PHONE] [e/CLIENT_EMAIL]…​`
```

Comment:
i think we discussed keeping prefixes to 1 letter if possible? so maybe p is project_id and c for contact_number or something along those lines